src/main/java/com/hillel/items_exchange/exceptionshandling/GlobalControllerExceptionHandler.java

```diff
@@ -0,0 +1,137 @@
+package com.hillel.items_exchange.exceptionshandling;
```

Comment:
Move current class to "exception.handler" package.

Reply:
Performed